articles/introduction-to-wireshark/index.md

```diff
@@ -0,0 +1,228 @@
+This will prompt you to another window, as seen below.  
+
+#### Fig 1.4 Network Interfaces 2
+![Network-Interfaces](/engineering-education/introduction-to-wireshark/wireshark-network-interfaces-2.jpg)
+
+Now select any of the interfaces you're presented with. In this example, we're going to explore the UDP Listener option.  
+Then click on the start button in the far left corner.  
+At this stage, you should be able to get an output like the one presented below.  
+
+#### Fig 1.5 Network Traffics
+![Network-Traffics](/engineering-education/introduction-to-wireshark/wireshark-live-capture.jpg)
+
+In the capture panel above, you're able to notice that network packet capturing is in sequential order, with each line representing each packet captured.  
+
+The information is displayed in tabular form, with rows and columns.  
+Each row represents the packet collected, while columns provide you with additional information such as time, protocols, length, et cetera. You can refer to the screenshot above.  
+
+Let's have a look at these columns and what type of information they provide us with.  
+* ```No``` - Represents a unique network packet sequence number. This can be used to identify a given packet.  
+* ``` Time ```- This is the time at which a particular packet was captured.  
+* ```Source ```- This represents where we are getting the packets from. This is denoted as Internet Protocols(IP Addresses).
+* ``` Destination```- This is used to represent the Internet Protocol(IP Address) where the packet is going.  
+* ```Protocol```- This refers to the protocol of the data you have captured. This could be ```TCP, ARP et cetera```
+* ```Length```- This is used to represent the size of the packet captured.
+* ``` Info```- This gives you additional information about the packet you have captured.  
+
+NOTE:- Each protocol is represented with its color scheme. In our case, for example, the ```TCP ``` protocol has a ```#cccccc``` background.  This helps the user to easily differentiate between these protocols.  
+
+
+Congratulations, you have been able to capture network packets, and that's how simple Wireshark is user friendly. 
+
+### Packet Details Panel:-
+Now that we can capture some data try to click on a single row, and you will notice that some data are being displayed on the immediate window.  
+
+#### Fig 1.5 Packet Details
+![Single-Packet-Details](/engineering-education/introduction-to-wireshark/wireshark-single-packet-details.jpg)
+On the highlighted protocol, click on it to get more details about this SSDP protocol as shown below:-  
+
+#### Fig 1.6 More on packet Details
+![Protocol-details](/engineering-education/introduction-to-wireshark/wireshark-protocol-details.jpg)
+
+The information above tells us more about the packet captured, including the device used. This data is very important, especially in system hacks, where they can be collected for forensics.  
+
+### Packet Bytes Panel
```

Comment:
Packet bytes panel

Do this for all headings. Only the first word should begin with a capital letter.